app/controllers/assignments_controller.rb

```diff
@@ -49,7 +49,7 @@ def assign_reviewers_randomly
 
   # Select an Available Article as a new Assignment for a user
   def claim
-    @claimed_assignment = Assignment.find(params[:assignment_id])
+    @claimed_assignment = Assignment.find(params[:id])
```

Comment:
Did you run the tests locally? This is a change I would expect to break things, as the parameter for this action from the `routes.rb` code is passed as `assignment_id`.